src/test/java/com/google/api/codegen/testsrc/common/library.proto

```diff
@@ -453,11 +429,15 @@ message Book {
 
 // A single book in the archives.
 message BookFromArchive {
+  option (google.api.resource) = {
+    type: "library.googleapis.com/BookFromArchive",
```

Comment:
Is it possible to rename this back to `ArchivedBook` to match (ignoring casing) what the library_gapic.yaml does?

Reply:
Yes will do